packages/astro/src/runtime.ts

```diff
@@ -335,7 +336,7 @@ async function createSnowpack(astroConfig: AstroConfig, options: CreateSnowpackO
   const mountOptions = {
     ...(existsSync(astroConfig.public) ? { [fileURLToPath(astroConfig.public)]: '/' } : {}),
     [fileURLToPath(frontendPath)]: '/_astro_frontend',
-    [fileURLToPath(projectRoot)]: '/_astro', // must be last (greediest)
+    [fileURLToPath(srcRoot)]: '/_astro/src', // must be last (greediest)
```

Comment:
`'/_astro' -> '/_astro/src'` is short-term fix since I know some people have hard-coded URLs and we want them to keep working.  If we're about to launch `Astro.ref()` to remove those hardcoding in your project, then I'd rather keep this bandaid around until we do that.

Reply:
I don’t see an issue with this, but if it’s short-term maybe put a TODO or comment?